Not compatible with cypress 13

This morning I upgraded to the latest cypress 13 so i can try the new cypress replay feature but i got npm error that it is not compatible and i need to downgrade to version 12. 

Thanks




npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: workflow@1.0.0
npm ERR! Found: cypress@13.0.0
npm ERR! node_modules/cypress
npm ERR!   dev cypress@"^13.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer cypress@"^4.x || ^5.x || ^6.x || ^7.x || ^8.x || ^9.x || ^10.x || ^11.x || ^12.x" from cypress-real-events@1.10.1
npm ERR! node_modules/cypress-real-events
npm ERR!   cypress-real-events@"^1.10.0" from the root project
npm ERR!

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "semantic-release": "semantic-release"
   },
   "peerDependencies": {
-    "cypress": "^4.x || ^5.x || ^6.x || ^7.x || ^8.x || ^9.x || ^10.x || ^11.x || ^12.x"
+    "cypress": "^4.x || ^5.x || ^6.x || ^7.x || ^8.x || ^9.x || ^10.x || ^11.x || ^12.x || ^13.x"
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.46.1",

